advocacy_docs/partner_docs/CohesityBackupandRecover/05-UsingCohesityConnectorAgentforEDBPostgres.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: 'Using Cohesity Connector Agent for EDB Postgres with EDB Postgres Advanced Server'
+description: 'Walkthrough of taking a backup and restore with the Cohesity Connector Agent for EDB Postgres scenario'
```

Comment:
Description needs to be simple and consistent with other guides. Change to: 

"Walkthroughs of multiple Cohesity Connector Agent for EDB Postgres usage scenarios"